catroid/src/org/catrobat/catroid/formulaeditor/FormulaEditorEditText.java

```diff
@@ -193,12 +193,17 @@ protected void onDraw(Canvas canvas) {
 		paint.setStrokeWidth(3);
 
 		Layout layout = getLayout();
+
 		if (layout != null) {
+
 			int line = layout.getLineForOffset(absoluteCursorPosition);
+
```

Comment:
3 lines of code, 4 empty lines... Remove *your* newly added empty lines.
Also https://wiki.openstack.org/wiki/GitCommitMessages#Things_to_avoid_when_creating_commits